crates/viewer/re_view_spatial/src/eye.rs

```diff
@@ -359,11 +372,7 @@ impl ViewEye {
         drag_threshold: f32,
         bounding_boxes: &SceneBoundingBoxes,
     ) -> bool {
-        let mut speed = match self.mode {
-            EyeMode::FirstPerson => 0.1 * bounding_boxes.current.size().length(), // TODO(emilk): user controlled speed
-            EyeMode::Orbital => self.orbit_radius,
-        };
```

Comment:
Previously the speed was chosen to adapt to the scene size. Small scenes: slow camera, big scenes: fast camera. This means the camera was always at least "okish".

The new code discards all that, and the new default speed of `1` may work for some scenes, but completely fail for others.

I'd like to have the default camera speed still be approximately the same as before. I see a few different ways to approach this:

A) The camera speed the user picks is relative to the old default (e.g. a multiplier, rather than an absolute speed)
B) The default camera speed is based on the old heuristic (instead of being 1)
C) The camera speed is the old heuristic until the user changes it (e.g. use `speed_override: Option<f32>`)

WDYT?